internal/pkg/service/buffer/store/slicestate/slicestate.go

```diff
@@ -18,12 +18,9 @@ const Opened State = "opened"
 // Waiting for the API nodes until they switch to the new slice.
 const Closing State = "closing"
 
-// Closed
-// The slice is ready for upload.
-const Closed State = "closed"
-
```

Comment:
Odstranil som slice `closed` state.

Predtym `closing->closed->uploading`.

Teraz `closing->uploading`.

Kedze je (dalsi PR) task na zatvorenie a task na upload, a `closed` mi to komplikovalo.